frontend/next.config.js

```diff
@@ -38,6 +38,10 @@ const defaultSecureHeaders = {
         HUBSPOT_FORMS_URL,
         PLAUSIBLE_URL,
         configs.API_URL,
+        "https://www.ebi.ac.uk/ols4/",
```

Comment:
@ashin-czi We can remove all these links now as the client is no longer making requests from the browser.